nexus/reconfigurator/planning/tests/output/planner_nonprovisionable_2_2a.txt

```diff
@@ -185,7 +185,7 @@ to:   blueprint 9f71f5d3-a272-4382-9154-6ea2e171a6c6
 
  REMOVED SLEDS:
 
-  sled 68d24ac5-f341-49ea-a92a-0381b52ab387 (was active):
+  sled 68d24ac5-f341-49ea-a92a-0381b52ab387 (state was unknown):
```

Comment:
`unknown` here is a side effect of the builder [removing the sled_state entry for fully-decommissioned sleds](https://github.com/oxidecomputer/omicron/blob/dcc0df384de6b1c708f715f66cc013c6fc3cde5c/nexus/reconfigurator/planning/src/blueprint_builder/builder.rs#L701-L712). We should stop doing that, but it's not directly relevant to this PR. (This changed because of the extra check for "look up the subnet of any active sled"; the test was claiming a sled was decommissioned in `PlanningInput` but not the parent blueprint, which is nonsense and caused the test to fail due to its data mucking rather than what it was actually trying to test.)